packages/design-system/src/components/web-components/ds-choice/ds-choice-list.test.tsx

```diff
@@ -1,7 +1,7 @@
 import { render, waitFor, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import './ds-choice-list';
-import '../ds-choice';
+import '.';
```

Comment:
I don't think I've seen this syntax before. Is it shorthand for importing `choice`?

Reply:
Whoops, that was vscode's automatic import refactor